public/app/plugins/panel/heatmap/panelcfg.cue

```diff
@@ -14,19 +14,112 @@
 
 package grafanaplugin
 
+import (
+	ui "github.com/grafana/grafana/packages/grafana-schema/src/common"
+)
+
 composableKinds: PanelCfg: {
 	lineage: {
 		seqs: [
 			{
 				schemas: [
 					{
+						HeatmapColorMode:  "opacity" | "scheme"     @cuetsy(kind="enum")
+						HeatmapColorScale: "linear" | "exponential" @cuetsy(kind="enum")
+						HeatmapColorOptions: {
+							mode?:    HeatmapColorMode // TODO: remove optional when https://github.com/grafana/cuetsy/issues/74 is fixed
```

Comment:
Both `mode` and `scale` had to be set to optional to be later overwritten in `types.ts`.